speech/snippets/transcribe_file_v2.py

```diff
@@ -56,10 +65,4 @@ def transcribe_file_v2(
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(
-        description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
-    )
-    parser.add_argument("project_id", help="GCP Project ID")
-    parser.add_argument("audio_file", help="Audio file to stream")
-    args = parser.parse_args()
-    transcribe_file_v2(args.project_id, args.audio_file)
+    transcribe_file_v2("resources/audio.wav")
```

Comment:
We should remove CLIs unless they're strictly necessary for the code sample within the region tags to work.